plugins/janus_streaming.c

```diff
@@ -80,8 +80,11 @@ audiortpmap = RTP map of the audio codec (e.g., opus/48000/2)
 audiofmtp = Codec specific parameters, if any
 audioskew = yes|no (whether the plugin should perform skew
 	analisys and compensation on incoming audio RTP stream, EXPERIMENTAL)
+minaudiorrinterval = minimum interval in ms between forwarding RTCP RR to audiortcpport 
```

Comment:
I don't think we should have this configuration for RR here: same thing for video (and RTSP). More on this below.